README.md

```diff
@@ -27,17 +25,35 @@ To run the site locally:
    ```
 4. Open http://localhost:8080
 
-### Accessiblity Scanning
+### Codespaces
+[Codespaces](https://github.com/features/codespaces) allows you to set up a development environment easily and without dependencies on a local machine.
+
+#### Start a GitHub Codespace
+1. From this repo, Click "Code" and then the "Codespaces" tab.
+2. Click the "+" icon, agree to the terms.
+3. You should then see the repo interface. At the bottom, check the branch you are currently viewing.
```

Comment:
Wondering if this would be a good spot to encourage the reader to create a new branch